docs/reference/cli/filters.md

```diff
@@ -83,6 +90,117 @@ This command will analyze your cluster Vulnerabilities through K8sGPT. Depending
 - Error: critical Vulnerability found ID: CVE-2019-8457 (learn more at: https://avd.aquasec.com/nvd/cve-2019-8457)
 ```
 
+## Prometheus
```

Comment:
I just realised that we should probably state the prerequisites for using the integrations maybe a top-level header:

## Prerequisites

For using the K8sGPT integrations please ensure that you have the latest version of the [K8sGPT CLI](../../getting-started/installation.md) installed. 
Also, please make sure that you are connected to a Kubernetes cluster. 

--------------
Once an integration requires additional tools etc. we can change it to make it integration specific?
Also, maybe we should highlight on top level that each integration will install resources into the cluster -- what do you think?

Reply:
Good call - added the Prerequisites header.

> Also, maybe we should highlight on top level that each integration will install resources into the cluster -- what do you think?

I'm hesitant here as the `prometheus` integration does not install any resources into the cluster.

Reply:
ah, I commented in that section that we should point out that people should have something installed vs. with Trivy, it's optional, and ideally they should not